docs/meshstack.how-to.integrate-meshplatform-azure-manually.md

```diff
@@ -81,6 +81,10 @@ All permissions left are therefore granted only via the Management Group hierarc
 
 # Rename pre-provisioned subscriptions, not required for Enterprise Enrollment
 "Microsoft.Subscription/rename/action"
+
+# The following permission is only required if you plan to use this principal for Azure Resource Group
+# integeration.
+"Microsoft.Resources/subscriptions/resourceGroups/write",
```

Comment:
You might have done that already but make sure that this change is also reflected in our Terraform service principal

Reply:
thats not so easy as we need some kind of switch to add this additional right. I need to look deeper into terraform how this might work.